### DISCUSSION TITLE
v3.6.0版本搜索功能在不同平台不可用的问题

### DISCUSSION THREAD

**Discussion by botforsms** [AlistGo/alist#2714] [Q&A]
索引 里面 用数据库模式已做好索引，
Windows下各浏览器好像**搜索功能**都好用，

但换系统就不行，

Android下Chrome浏览器搜索功能不好用，微信自带浏览器也不好用，

MAC下Safari、Chrome、Edge的搜索功能都不好用，

版本是v3.6.0，

大家有遇到这个问题吗？还是就我哪里有问题？

**Comment by fastxteam** [comment]
mac搜索都没搜索到过，其他的还没测试
